pkg/kubelet/apis/deviceplugin/v1alpha/api.proto

```diff
@@ -13,32 +13,19 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 option (gogoproto.goproto_unrecognized_all) = false;
 
-
-// Registration is the service advertised by the Kubelet
-// Only when Kubelet answers with a success code to a Register Request
-// may Device Plugins start their service
-// Registration may fail when device plugin version is not supported by
-// Kubelet or the registered resourceName is already taken by another
-// active device plugin. Device plugin is expected to terminate upon registration failure
-service Registration {
-	rpc Register(RegisterRequest) returns (Empty) {}
-}
-
```

Comment:
This is an incompatible API change. I think we will need a new API directory. Is it too early to create a deviceplugin/v1beta1 directory? I know we haven't flipped the feature gate flag yet, but it seems generally api version is a pre-requirement for feature gate version. Perhaps we should discuss this on this week's RW meeting?

Reply:
Thanks, this is an important question. Unfortunately missed the opportunity to discuss it at last meeting. Lets target for next meeting and meanwhile will try to discuss and get opinion on slack and list.